.tekton/basic-component-pipeline.yaml

```diff
@@ -172,12 +172,45 @@ spec:
         - name: basic-auth
           workspace: git-auth
 
+    - name: determine-image-tag
+      params:
+        - name: image
+          value: $(params.output-image)
+      runAfter:
+      - clone-repository
```

Comment:
[nit] A small comment here that the task must run on a freshly cloned repository to prevent seeing any changes from other tasks would be helpful.

Reply:
Done in https://github.com/stackrox/stackrox/pull/10749/commits/e2a34d52edee3087903c2817ab948c07eb19623f